lib/Conversion/VerifToSMT/VerifToSMT.cpp

```diff
@@ -274,6 +274,16 @@ struct VerifBoundedModelCheckingOpConversion
       }
     }
 
+    // TODO: this can be removed once we have a way to associate reg ins/outs
+    // with clocks
+    if (clockIndexes.size() > 1)
+      return op->emitError(
+          "only modules with one clock are currently supported");
+
+    // If we have no clocks then num_regs has to be 0
+    if (clockIndexes.size() == 0 && numRegs != 0)
+      return op->emitError("num_regs is non-zero but found no clock");
```

Comment:
That looks like something that should be verified by the BMC op, right?

Reply:
Yeah good point, I'll drop it here and add it to the verifier in another PR